# Contains extrinsics information for a set of frames relative to a reference

string[] frameNames
geometry_msgs/Pose[] extrinsics
